### DISCUSSION TITLE
Order of planned milestones?

### DISCUSSION THREAD

**Discussion by landerss1** [ory/kratos#1679] [Q&A] [answered]
We logged an [issue ](https://github.com/ory/kratos/issues/834) in November last year, pertaining to i18n support in email templates. I can see that it is planned for the "Incubating Release", just as a couple of other related issues. What is the order of the planned releases?

Is it:

- v0.8.0-alpha.1
- v0.9.0-alpha.1
- v0.10.0-alpha.1
- Incubating Release
- Stable Release

Is this right?

**Comment by vinckr** [comment, marked as answer]
Yes, the order of planned releases is right! 
The content of milestones might always change a bit, as we prioritize the issues according to community demands.
Looking at the issue, it already shows a good path for implementation if you happen to have some resources to work on it!

**Reply by landerss1** [reply]
Thanks for the clarification!

Depends on the prioritization from our own Product Manager...
